frontends/election-manager/src/components/card_programming_prompt.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const InsertCardImage = styled.img`
+  margin-right: auto;
+  margin-left: auto;
+  height: 20vw;
+`;
+
+interface CardProgrammingPromptProps {
+  cardType: 'election' | 'superAdmin';
+}
+
+export function CardProgrammingPrompt({
+  cardType,
+}: CardProgrammingPromptProps): JSX.Element {
+  return (
+    <React.Fragment>
+      <h2>{cardType === 'election' ? 'Election' : 'Super Admin'} Cards</h2>
+      <InsertCardImage src="/assets/insert-card.svg" alt="" />
```

Comment:
noticing that this image has some white-filled parts and some transparent parts. looks a bit weird

Reply:
Yes, both Drew and I noticed this as well! https://github.com/votingworks/vxsuite/pull/1993#issuecomment-1162739340

We're waiting to hear back from @beausmith. In the meantime, I opened https://github.com/votingworks/vxsuite/issues/2022 to make sure we don't lose track of this